<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>View</class>
 <widget class="QMainWindow" name="View">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>View</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2" rowstretch="0,0,0,0,0,0,0,0,0,0,0" columnstretch="0,0,0,0,0,0,0,0,0,0" rowminimumheight="0,0,0,0,0,0,0,0,0,0,0" columnminimumwidth="0,0,0,0,0,0,0,0,0,0">
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
    <property name="horizontalSpacing">
     <number>-1</number>
    </property>
    <property name="verticalSpacing">
     <number>4</number>
    </property>
    <item row="8" column="6" colspan="2">
     <widget class="QPushButton" name="digit_0">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="6" column="4">
     <widget class="QPushButton" name="oper_percent">
      <property name="text">
       <string>%</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="func_sin">
      <property name="text">
       <string>sin</string>
      </property>
     </widget>
    </item>
    <item row="5" column="9">
     <widget class="QPushButton" name="oper_minus">
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="QPushButton" name="func_log">
      <property name="text">
       <string>log</string>
      </property>
     </widget>
    </item>
    <item row="5" column="7">
     <widget class="QPushButton" name="digit_5">
      <property name="text">
       <string>5</string>
      </property>
     </widget>
    </item>
    <item row="4" column="6">
     <widget class="QPushButton" name="digit_7">
      <property name="text">
       <string>7</string>
      </property>
     </widget>
    </item>
    <item row="6" column="8">
     <widget class="QPushButton" name="digit_3">
      <property name="text">
       <string>3</string>
      </property>
     </widget>
    </item>
    <item row="5" column="8">
     <widget class="QPushButton" name="digit_6">
      <property name="text">
       <string>6</string>
      </property>
     </widget>
    </item>
    <item row="4" column="8">
     <widget class="QPushButton" name="digit_9">
      <property name="text">
       <string>9</string>
      </property>
     </widget>
    </item>
    <item row="6" column="7">
     <widget class="QPushButton" name="digit_2">
      <property name="text">
       <string>2</string>
      </property>
     </widget>
    </item>
    <item row="5" column="4">
     <widget class="QPushButton" name="oper_mod">
      <property name="text">
       <string>mod</string>
      </property>
     </widget>
    </item>
    <item row="8" column="8">
     <widget class="QPushButton" name="oper_point">
      <property name="text">
       <string>.</string>
      </property>
     </widget>
    </item>
    <item row="8" column="4">
     <widget class="QPushButton" name="variable_x">
      <property name="text">
       <string>x</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QPushButton" name="EXP">
      <property name="text">
       <string>EXP</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QPushButton" name="func_ln">
      <property name="text">
       <string>ln</string>
      </property>
     </widget>
    </item>
    <item row="4" column="9">
     <widget class="QPushButton" name="oper_mult">
      <property name="text">
       <string>*</string>
      </property>
     </widget>
    </item>
    <item row="5" column="3">
     <widget class="QPushButton" name="func_atan">
      <property name="text">
       <string>atan</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QPushButton" name="set_x">
      <property name="text">
       <string>set_x</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="6" colspan="4">
     <widget class="QLineEdit" name="lineVar">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="focusPolicy">
       <enum>Qt::ClickFocus</enum>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="placeholderText">
       <string>Value of x = 0</string>
      </property>
     </widget>
    </item>
    <item row="3" column="7">
     <widget class="QPushButton" name="clean_D">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>D</string>
      </property>
     </widget>
    </item>
    <item row="8" column="3">
     <widget class="QPushButton" name="variable_e">
      <property name="text">
       <string>e</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="10">
     <widget class="QLineEdit" name="lineEdit">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="toolTipDuration">
       <number>0</number>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QPushButton" name="oper_lb">
      <property name="text">
       <string>(</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0" colspan="10">
     <widget class="QCustomPlot" name="graf" native="true"/>
    </item>
    <item row="8" column="2">
     <widget class="QPushButton" name="variable_p">
      <property name="text">
       <string>π</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QPushButton" name="func_cos">
      <property name="text">
       <string>cos</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="credit">
      <property name="text">
       <string>credit</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QPushButton" name="deposit">
      <property name="text">
       <string>deposit</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QPushButton" name="func_acos">
      <property name="text">
       <string>acos</string>
      </property>
     </widget>
    </item>
    <item row="8" column="9">
     <widget class="QPushButton" name="equal">
      <property name="text">
       <string>=</string>
      </property>
     </widget>
    </item>
    <item row="6" column="9">
     <widget class="QPushButton" name="oper_plus">
      <property name="text">
       <string>+</string>
      </property>
     </widget>
    </item>
    <item row="3" column="8">
     <widget class="QPushButton" name="oper_unopm">
      <property name="text">
       <string>±</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QPushButton" name="func_asin">
      <property name="text">
       <string>asin</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="button_replot">
      <property name="text">
       <string>replot</string>
      </property>
     </widget>
    </item>
    <item row="3" column="4">
     <widget class="QPushButton" name="oper_rb">
      <property name="text">
       <string>)</string>
      </property>
     </widget>
    </item>
    <item row="6" column="3">
     <widget class="QPushButton" name="oper_pow">
      <property name="text">
       <string>pow</string>
      </property>
     </widget>
    </item>
    <item row="5" column="6">
     <widget class="QPushButton" name="digit_4">
      <property name="text">
       <string>4</string>
      </property>
     </widget>
    </item>
    <item row="3" column="9">
     <widget class="QPushButton" name="oper_div">
      <property name="text">
       <string>÷</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="button_graf">
      <property name="text">
       <string>graf</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="4">
     <widget class="QPushButton" name="func_sqrt">
      <property name="text">
       <string>√</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QPushButton" name="clean_AC_x">
      <property name="styleSheet">
       <string notr="true">QPushButton {
	font: 13pt &quot;Arial&quot;;
	color: rgb(216, 62, 44);
}</string>
      </property>
      <property name="text">
       <string>AC_x</string>
      </property>
     </widget>
    </item>
    <item row="6" column="6">
     <widget class="QPushButton" name="digit_1">
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="ANS">
      <property name="text">
       <string>ANS</string>
      </property>
     </widget>
    </item>
    <item row="3" column="6">
     <widget class="QPushButton" name="clean_C">
      <property name="styleSheet">
       <string notr="true">QPushButton {
	color: red;
}</string>
      </property>
      <property name="text">
       <string>C</string>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QPushButton" name="func_tan">
      <property name="text">
       <string>tan</string>
      </property>
     </widget>
    </item>
    <item row="4" column="7">
     <widget class="QPushButton" name="digit_8">
      <property name="text">
       <string>8</string>
      </property>
     </widget>
    </item>
    <item row="3" column="5">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Preferred</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="10" column="0" colspan="10">
     <widget class="QFrame" name="frame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>580</width>
        <height>30</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QDoubleSpinBox" name="x_begin">
       <property name="geometry">
        <rect>
         <x>5</x>
         <y>2</y>
         <width>110</width>
         <height>25</height>
        </rect>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="minimum">
        <double>-1000000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000000.000000000000000</double>
       </property>
       <property name="value">
        <double>-10.000000000000000</double>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="x_end">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>2</y>
         <width>110</width>
         <height>25</height>
        </rect>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="minimum">
        <double>-1000000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000000.989999999990687</double>
       </property>
       <property name="value">
        <double>10.000000000000000</double>
       </property>
      </widget>
      <widget class="QLabel" name="label_x">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>2</y>
         <width>50</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;= x &lt;=</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="y_begin">
       <property name="geometry">
        <rect>
         <x>295</x>
         <y>2</y>
         <width>110</width>
         <height>25</height>
        </rect>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="minimum">
        <double>-1000000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000000.000000000000000</double>
       </property>
       <property name="value">
        <double>-10.000000000000000</double>
       </property>
      </widget>
      <widget class="QLabel" name="label_y">
       <property name="geometry">
        <rect>
         <x>410</x>
         <y>2</y>
         <width>50</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;= y &lt;=</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="y_end">
       <property name="geometry">
        <rect>
         <x>465</x>
         <y>2</y>
         <width>110</width>
         <height>25</height>
        </rect>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="minimum">
        <double>-1000000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000000.000000000000000</double>
       </property>
       <property name="value">
        <double>10.000000000000000</double>
       </property>
      </widget>
     </widget>
    </item>
    <item row="1" column="5">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>30</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="3" colspan="2">
     <widget class="QLineEdit" name="lineVarError">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>15</pointsize>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
	font: 15pt &quot;Arial&quot;;
	color: rgb(127, 0, 0);
    background-color:rgb(50, 50, 50);
    border: 0px solid rgb(55, 55, 55);
}</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
